tests/utils/src/main/java/org/keycloak/tests/utils/SimpleHttp.java

```diff
@@ -0,0 +1,32 @@
+package org.keycloak.tests.utils;
+
+import org.apache.http.client.HttpClient;
+import org.keycloak.connections.httpclient.HttpClientProvider;
+
+public class SimpleHttp extends org.keycloak.broker.provider.util.SimpleHttp {
```

Comment:
Instead of adding a whole new class, can't we add overloaded static methods to the original class e.g.

```java
    protected static SimpleHttp doGet(String url, HttpClient client) {
        return new SimpleHttp(url, "GET", client, HttpClientProvider.DEFAULT_MAX_CONSUMED_RESPONSE_SIZE);
    }
```